worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerFactory.java

```diff
@@ -318,9 +317,9 @@ public void determineForwardingAction(TaskMessage proxiedTaskMessage, TaskInform
                 f.setFailureTime(new Date());
                 f.setFailureSource(getWorkerName(proxiedTaskMessage));
 
-                final byte[] taskData = proxiedTaskMessage.getTaskData();
+                final Object taskData = proxiedTaskMessage.getTaskData();
```

Comment:
But `.getTaskData()` still returns a `byte[]`.  What is being achieved by this?

Reply:
Restored